src/test_helpers/bart_test_helper.h

```diff
@@ -0,0 +1,122 @@
+#ifndef BART_TEST_HELPERS_BART_TEST_HELPER_H_
+#define BART_TEST_HELPERS_BART_TEST_HELPER_H_
+
+#include <string>
+#include <memory>
+#include <fstream>
+#include <sys/stat.h>
+#include <ctime>
+#include <iomanip>
+
+#include <deal.II/base/logstream.h>
+#include <deal.II/base/utilities.h>
+#include <deal.II/base/mpi.h>
+#include "gtest/gtest.h"
+
+#include "gold_stream_evaluator.h"
+
+namespace btest {
+//! This class provides a test framework for BART using gTest.
+/*!
+ 
+Functionality includes:
+
+(1) Managing a filestream resource for the dealii log.
+
+(2) Running a gold file test to compare two files.
+ 
+ \author Joshua Rehak
+ \date 2018/2
+ */
+class BartTestHelper {
+ public:
+  //! Default constructor, default values `report = true`, `gold_files_directory = "test_data/"`
+  BartTestHelper();
+  //! Constructor specifying if a report will be generated and location of gold files.
```

Comment:
An empty line separating each declaration would be better for readability. Human eyes are not good at looking at things staying too close.